### DISCUSSION TITLE
Getting Error while running Artillery Test with Jenkins

### DISCUSSION THREAD

**Discussion by shiv-vsics** [artilleryio/artillery#3287] [Artillery]
Hi @hassy / @bernardobridge 

I was running Artillery Test with Jenkins but its giving me error at the step of installing artillery via my pipeline scirpt.
Error looks as below:

**+ npm install -g artillery
npm error code ETARGET
npm error notarget No matching version found for @smithy/protocol-http@^4.1.0.**

I can see this smithy version is available. This failure is happening from today itself, it was working fine till yesterday. Any update has been happened ? do i need to make any update ? Please suggest.

**Comment by bernardobridge** [comment]
Hey @shiv-vsics 👋 

A couple of questions:
- If you install a specific older version of artillery (e.g. 2.0.17 or 2.0.16) using `npm install -g artillery@2.0.16`, do you still get the error?
- Do you use a private registry (e.g. Artifactory)?

It's more likely the error has something to do with your setup, as we don't list `@smithy/protocol-http@^4.1.0` as a dependency, it's a transient dependency of other packages and it doesn't explicitly require ^4.1.0.
